rules/do-you-use-global-list/rule.md

```diff
@@ -17,7 +17,7 @@ Global list could be referenced in multiple work item types, if you are using th
 
 <!--endintro-->
 
-```
+```xml
 &lt;FIELD
```

Comment:
Hey @tombui99 

Since these are inside code tags, we should use the actual characters `<` and `>` instead of `&lt;` and `&gt;``

1. Please fix all

Reply:
Hi @tiagov8 

Thanks for the comment!

> Fix all

Done - special characters turned into normal